authentication/wildfly/pom.xml

```diff
@@ -47,7 +47,8 @@
         
         <wildfly.home>${project.build.directory}/wildfly</wildfly.home>
         <debugJvmArgs/>
-        <wildfly.skip>false</wildfly.skip>
+        <provision.skip>false</provision.skip>
+        <configure.skip>false</configure.skip>
```

Comment:
Minor nit. I've been starting to lean towards using the properties defined by the plugin itself. So we could use `galleon.skip` for provisioning and `wildfly.skip` for the maven plugin. 

Again not critical at all. It just makes it easier to know what the properties are across projects IMO.